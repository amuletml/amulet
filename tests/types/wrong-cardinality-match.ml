type t = A of int

let f = function
| A -> ()
