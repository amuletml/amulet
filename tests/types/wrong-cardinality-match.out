wrong-cardinality-match.ml[4:3 ..4:3]: error (E2042)
  │ 
4 │ | A -> ()
  │   ^
  Constructor takes 1 arguments but was given none
